include/swift/AST/DiagnosticsSema.def

```diff
@@ -719,11 +719,10 @@ ERROR(sema_opening_import,Fatal,
 ERROR(serialization_load_failed,Fatal,
       "failed to load module '%0'", (StringRef))
 ERROR(module_interface_build_failed,Fatal,
-      "failed to %select{build module '%1' from its module interface|verify "
-      "module interface of '%1'}0; %select{the compiler that produced it, "
-      "'%3', may have used features that aren't supported by this compiler, "
-      "'%4'|it may have been damaged or it may have triggered a bug in the "
-      "Swift compiler when it was produced}2",
+      "failed to %select{build module '%1' for importation|"
+      "verify module interface of '%1'}0 due to the errors above; "
+      "the textual interface may be broken by project issues"
+      "%select{, incompatible compilers (the producer '%3' and this compiler '%4')|}2 or a compiler bug",
```

Comment:
I'm a little concerned that "may be broken by...incompatible compilers" might imply that we *know* the compilers are incompatible. It might be better to say something like "differences between compilers".

Reply:
I agree! and updated the PR.